securedrop-admin

```diff
@@ -128,8 +128,11 @@ def install_securedrop(args):
 
 def run_tails_config(args):
     """Configure Tails environment post SD install"""
-    subprocess.check_call(['sudo',
-                            os.path.join(SD_DIR, 'tails_files/install.sh')])
+    activate_venv()
+    subprocess.check_call([os.path.join(ANSIBLE_PATH, 'securedrop-tails.yml'),
```

Comment:
I'm actually not much of a fan of the executable playbook pragma, simply because it's not common in the Ansible community. Added it here to be consistent with the tooling changes as of #1781.